webpack.common.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+const path = require('path');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const webpack = require('webpack');
+
+/*::
+type Target = {|
+    name: string, // the name of output JS/CSS
+    entry: string, // the path to the entry point
+    library?: string // the name of the exported module
+|};
+*/
+
+/**
+ * List of targets to build
+ */
+const targets /*: Array<Target> */ = [
+    {
+        name: 'katex',
+        entry: './katex.webpack.js',
+        library: 'katex',
+    },
+    {
+        name: 'contrib/auto-render',
+        entry: './contrib/auto-render/auto-render.js',
+        library: 'renderMathInElement',
+    },
+    {
+        name: 'contrib/copy-tex',
+        entry: './contrib/copy-tex/copy-tex.js',
+    },
+    {
+        name: 'contrib/mathtex-script-type',
+        entry: './contrib/mathtex-script-type/mathtex-script-type.js',
+    },
+];
+
+/**
+ * Create a webpack config for given target
+ */
+function createConfig(target /*: Target */, dev /*: boolean */,
+        minimize /*: boolean */) /*: Object */ {
+    const cssLoader = {
+        loader: 'css-loader',
+        options: {
+            minimize, // cssnano
```

Comment:
What's `cssnano`?

Reply:
@kevinbarabash [`cssnano`](https://github.com/ben-eb/cssnano) is a CSS minifier equivalent to `clean-css` and it is included in the `css-loader`, which can be enabled by passing `minimize: true` to its options.